src/components/templates/Playa/VenuePreview.tsx

```diff
@@ -159,11 +160,12 @@ const VenuePreview: React.FC<VenuePreviewProps> = ({
       .then(function (array) {
         const futureEvents = array.docs
           .map((doc) => doc.data())
-          .filter((event) => event.start_utc_seconds > nowSeconds)
-          .sort((a, b) => a.start_utc_seconds - b.start_utc_seconds);
+          .filter((event) => event.start_utc_seconds > nowSeconds);
```

Comment:
I guess we might want that to be in the `utils/event.ts` -> `isFutureEvent()`

Reply:
I struggled with this a bit due to the `array.docs` part. It's a good idea to move the function to `utils/event.ts`, but seems a bit out of scope for this PR. I only added the `sortEventsByStartUtcSeconds` function to be consistent with the change elsewhere in this PR

Reply:
Given it's a 1 liner to implement, it would be nice to fix it up in this PR. Specifically this part, using `date-fns`:

```typescript
(event) => event.start_utc_seconds > nowSeconds
```

We already have `isEventLiveOrFuture` there though.. which I assume would solve this need, so we should just be using that.

Reply:
Thanks for the tip — done: 20f39aee52b31f9839f5a9773aafa974ecdaec0b